tests/functional/u/unused/unused_private_member.py

```diff
@@ -309,3 +309,13 @@ class Foo:
 
     def method(self):
         print(self.__class__.__ham)
+
+
+class TypeSelfCallInMethod:
+    """Regression test for issue 5569"""
+    @classmethod
+    def b(cls) -> None:
+        cls.__a = ''  # [unused-private-member]
```

Comment:
Unrelated but shouldn't this raise `attribute-defined-outside-init`?

Reply:
Ah, but `__init__()` is for instance attributes, not class attributes, no?

Reply:
Ah, of course! Although I think patterns such as this might equally create trouble with uninitialised attributes down the line 😄